### PR TITLE
Pileometh

### PR DESCRIPTION
**Pull request by AGoepfert**
Cameron solved my main problem and I made some tiny changes! It should work without any problems! But I am not sure if the tool_dep is working.

### CODE REVIEW

File: tools/pileometh/test-data/cg100.fa.fai [hunk 1]
```diff
@@ -0,0 +1,837 @@
+
```
Comment (bgruening) [inline tools/pileometh/test-data/cg100.fa.fai]: This file looks strange and it not a `fai` file. It looks like you tried to download it from github?

**Comment by AGoepfert** [comment]
My mistake! I didn't commit the deleted files - Is this the reason why they were visible for you?

**Comment by bgruening** [comment]
Yes this can be. Can you please rebase against master? Thanks!

**Comment by AGoepfert** [comment]
Everything looks perfect! and unnecessar files are not there anymore!

**Pull request merged**